themes/jb/assets/css/layout/show-slider.scss

```diff
@@ -1,7 +1,7 @@
 
 .show-img {
- height: 168px !important;
- width: 300px !important
+//  height: 168px !important;
+//  width: 300px !important
 }
```

Comment:
Any reason for removing this? I know the show's images are about that size anyway, but I wonder if we leave the sizing there just in case a new show gets added at some point in the future and the size does not match the rest.

What do you think?

Reply:
Because it actually creates dead pixel space and will cause the alignment to look off. The images will fill their containers. Unless the aspect ratio changes (which would be a bigger issues) I don't see much risk.

Reply:
I just think back to the issue we had with profile pictures (#377). Just want to be prepared for as much as possible. Looking at the images, they seem to resize themselves a few pixels larger:
![image](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/assets/34293941/dac6402e-10ff-4a98-88ac-f91996a11b3a)

Maybe just update the comments to match what the images are when they fill the containers? That way if we ever need to bring them back, it'll be ready to go.